packages/zowe-explorer-api/src/vscode/ZoweVsCodeExtension.ts

```diff
@@ -45,24 +45,6 @@ export class ZoweVsCodeExtension {
         return undefined;
     }
 
-    /**
-     * Reveal an error in VSCode, and log the error to the Imperative log
-     *
-     */
-    public static showVsCodeMessage(message: string, severity: MessageSeverityEnum, logger: IZoweLogger): void {
```

Comment:
instead of removing them, we could 
```typescript
/**
 * @deprecated Please use Gui.showMessage
 */
```

Then, make this a wrapper of `Gui.showMessage`